src/Tgstation.Server.Api/Models/DiscordConnectionStringBuilder.cs

```diff
@@ -42,9 +47,12 @@ public DiscordConnectionStringBuilder(string connectionString)
 			if (splits.Length < 2 || !Enum.TryParse<DiscordDMOutputDisplayType>(splits[1], out var dMOutputDisplayType))
 				dMOutputDisplayType = DiscordDMOutputDisplayType.Always;
 			DMOutputDisplay = dMOutputDisplayType;
+			if (splits.Length < 3 || !bool.TryParse(splits[2], out bool basedMeme))
+				basedMeme = false;
```

Comment:
Use numbers instead of the words true/false to maintain parity with IrcConnectionStringBuilder please

https://github.com/tgstation/tgstation-server/blob/64927f4ede73fbf36bdd9c8e9044db1c36dd0e82/src/Tgstation.Server.Api/Models/IrcConnectionStringBuilder.cs#L77-L78

Reply:
God damnit my BYOND habits are killing me